python/xorbits/pandas/pandas_adapters/core.py

```diff
@@ -173,7 +173,7 @@ def _collect_pandas_index_members():
     _collect_pandas_cls_members(pd.Index, DataType.index)
 
 
-def _collect_pandas_module_members() -> Dict[str, Any]:
+def collect_pandas_module_members() -> Dict[str, Any]:
```

Comment:
Like I said, we can cache the result of this function to avoid using global variables.

Reply:
> Like I said, we can cache the result of this function to avoid using global variables.

Since there are similar usages in the NumPy adapter, how about removing the global variables all at once in a following PR?

Reply:
Good idea, we can do them all in the following PR.